src/test/field/index.spec.ts

```diff
@@ -14,11 +14,31 @@ describe('Field', () => {
     @ObjectType()
     class Foo {
       @Field() bar: string = 'baz';
+      @Field() undef: boolean = undefined;
```

Comment:
I'd say this test is out of scope of `resolving default value`. It's rather another test case like `Resolves with edge-case default values`